### PR TITLE
Created apollo service and added new dependencies

### PR DESCRIPTION
**Pull request by ecl1pseo**

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -3,25 +3,25 @@ import { Provider } from 'react-redux';
...
+import { ApolloServiceProvider } from '../../services/ApolloService/ApolloServiceContext';
+import { client } from '../../services/ApolloService/ApolloService';
 import store from '../../store/store';
 
 // import Routes from '../routes';
 
 const App = () => (
   <Provider store={store}>
-    <GraphqlServiceProvider value={graphqlService}>
+    <ApolloServiceProvider value={client}>
```
Comment (horondi) [inline src/components/App/App.js:16]: I don't think that we need to use `ApolloServiceProvider`. We are going to dispatch actions by ourselves and handle them in sagas. The only thing that we need from `apollo` is actual client.

File: src/components/Test/index.js [hunk 2]
```diff
@@ -1,40 +1,25 @@
...
   const handler = () => {
-    graphqlService
-      .getItems(
-        `query{
-        categories{
-          categoryCode,
-          _id
-          name{
-            value
+    client
```
Comment (horondi) [inline src/components/Test/index.js:7]: I think it is better to move it to the `redux-saga` definition. Please:
1. Install `redux-saga` and setup it in application
2. Define saga and actions for fetching categories
3. Dispatch action from component

File: package.json [hunk 3]
```diff
@@ -3,26 +3,32 @@
...
+    "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.1",
+    "react-scripts-config": "^1.0.17-13",
     "react-scroll-up-button": "^1.6.4",
     "react-star-ratings": "^2.3.0",
     "redux": "^4.0.5",
-    "redux-thunk": "^2.3.0",
-    "react-scripts-config": "^1.0.17-13"
+    "redux-saga": "^1.1.3",
+    "redux-thunk": "^2.3.0"
```
Comment (horondi) [inline package.json:31]: Please, remove `redux-thunk`, we don't need it anymore

File: src/services/ApolloService/ApolloService.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import ApolloClient from 'apollo-boost';
+
+export const client = new ApolloClient({
+  uri: 'http://localhost:5000/'
```
Comment (VolodymyrTrach) [inline src/services/ApolloService/ApolloService.js:4]: rename uri to base_url (or something like this) and move it to .env. or import it from .env

Comment (ecl1pseo) [reply src/services/ApolloService/ApolloService.js]: moved to .env variable REACT_APP_BASE_URI

File: src/components/App/App.js (hunk 1 above)
Comment (ecl1pseo) [reply src/components/App/App.js]: Deleted ApolloProvider

File: package.json (hunk 3 above)
Comment (ecl1pseo) [reply package.json]: Removed redux-thunk dependency

File: src/components/Test/index.js (hunk 2 above)
Comment (ecl1pseo) [reply src/components/Test/index.js:7]: Installed redux-saga. Task with defining saga fot test component will take another teammate

**Pull request closed without merging**